corehq/apps/cloudcare/views.py

```diff
@@ -605,3 +607,24 @@ def _fail(error):
         })
     cloudcare_state = json.dumps(state)
     return HttpResponseRedirect(reverse(FormplayerMain.urlname, args=[domain]) + "#" + cloudcare_state)
+
+
+@login_and_domain_required
+@require_POST
+def api_histogram_metrics(request, domain):
+    request_dict = request.POST
+
+    metric_name = request_dict.get("metrics")
+    try:
+        duration = float(request_dict.get("responseTime"))
+    except ValueError:
+        duration = None
```

Comment:
shouldn't this be treated as an error?

Reply:
This is added as a fail-safe but if checked, from frontend data will always be sent as a float string containing duration in ms as done [here](https://github.com/dimagi/commcare-hq/pull/33678/files/c933dc44eacb59e296879abb49981c1e1e582dcc#diff-e9a325527828d83ca273a6ebef7ca67882d9dee8484486bbfcea88215b8c8531R172). 
Should I remove this try/except instead as ideally this exception will not happen?

Reply:
I think so, then we'd get a sentry notification if this broke down the road.

Reply:
Removed try/except